pylint/checkers/typecheck.py

```diff
@@ -733,15 +735,25 @@ class TypeChecker(BaseChecker):
                 "no-member and other checks for the rest of the inferred objects.",
             },
         ),
+        (
+            "mixin-class-rgx",
+            {
+                "default": ".*mixin",
+                "type": "regexp",
+                "metavar": "<regexp>",
+                "help": "Indicates which classes are considered mixins if "
+                "ignore-mixin-members is True. Defaults to ending in Mixin "
+                "(case insensitive).",
+            },
+        ),
         (
             "ignore-mixin-members",
             {
                 "default": True,
                 "type": "yn",
                 "metavar": "<y_or_n>",
-                "help": 'Tells whether missing members accessed in mixin \
-class should be ignored. A mixin class is detected if its name ends with \
-"mixin" (case insensitive).',
+                "help": "Tells whether missing members accessed in mixin "
+                "class should be ignored.",
```

Comment:
Maybe we could have left: "A mixin class is detected if its name match the regex defined by mixin_class_rgx" or something similar.